tests/framework/CodeInterfaceTests/generic_parallel.xml

```diff
@@ -6,12 +6,13 @@
     <created>2017-08-24</created>
     <classesTested>Models.Code.GenericCode, Files</classesTested>
     <description>
-        Tests using "Files" in an internalParallel run (which tests pickling and unpickling them).
+        Tests using &quot;Files&quot; in an internalParallel run (which tests pickling and unpickling them).
```

Comment:
&quot?

Reply:
It appears that when the conversion script runs, it doesn't write out the quotation marks well.  Interesting.  I'll get rid of them so it doesn't come up again.